bip-0118.mediawiki

```diff
@@ -175,9 +175,21 @@ Following this recommendation may require additional script branches, which may
 
 == Deployment ==
 
-TODO
+This BIP is deployed as a soft fork subsequent to the deployment of [[bip-0340.mediawiki|BIP 340]], [[bip-0341.mediawiki|BIP 341]] and [[bip-0342.mediawiki|BIP 342]].
 
-This may be deployed as a soft-fork either concurrent with, or subsequent to the deployment of [[bip-0340.mediawiki|BIP 340]], [[bip-0341.mediawiki|BIP 341]] and [[bip-0342.mediawiki|BIP 342]].
+For Bitcoin signet, this BIP can be always active for testing purposes.
+
+For Bitcoin mainnet and testnet3, this BIP should be deployed by "version bits" [[bip-0009.mediawiki|BIP 9]] with the name "anyprevout" and a bit to be determined, using the same activation mechanism as [[bip-0341.mediawiki|BIP 341]].
```

Comment:
I don't know how much longer testnet3 will be around: https://github.com/bitcoin/bitcoin/pull/31974